packages/plugin-ext/src/common/plugin-protocol.ts

```diff
@@ -663,6 +665,18 @@ export interface Keybinding {
     win?: string;
 }
 
+/**
+ * Keybinding contribution
+ */
+export interface Keybinding {
```

Comment:
@vinokurig 
I guess the interface was added by mistake, could you [take a look](https://github.com/eclipse-theia/theia/blob/master/packages/plugin-ext/src/common/plugin-protocol.ts#L702-L721) 
I created a PR to remove it https://github.com/eclipse-theia/theia/pull/9374